Navigation/Screens/AddExpenses.tsx

```diff
@@ -1,69 +1,44 @@
 import React from "react";
-import { Text ,TextInput} from "react-native";
 import { Button } from "react-native";
 import { View } from "react-native";
-import SelectDropdown from "react-native-select-dropdown";
-import DatePicker from "react-native-date-picker";
 import { useState } from "react";
 import { StyleSheet } from "react-native";
+import InputForm from "../../components/InputForm";
+import DropDown from "../../components/DropDown";
+import DateInput from "../../components/DateInput";
+import ExpenseContext from "../../Context/ExpenseContext";
+import { useContext } from "react";
 
-export default function AddExpenses(){
-    const categories =["Food","Transportation","Utilities"]
-    const [date,setDate]=useState(new Date())
-    const [open,setOpen]= useState(false)
-    const [expense,setExpense] = useState({title:'',recipient:'',amount:'',category:'',date:''})
+
+export default function AddExpenses(props: { navigation: { navigate: (arg0: string) => void; }; }){
+   //console.log(Date.now())
+    const {expense,setExpense}:any = useContext(ExpenseContext)
+    const [addExpense,setAddExpense] = useState({id:'',title:'',recipient:'',amount:'',category:'',date:''})
+    type T = keyof typeof addExpense
```

Comment:
you can name the type better and you can can keep this outside the function scope